frontend/pages/index.vue

```diff
@@ -1,3 +1,132 @@
 <template>
-  <h1>Home Page</h1>
+  <v-container fluid>
+    <v-row>
+      <v-col cols="12">
+        <h1 class="text-h4 ml-4"> Dashboard </h1>
+      </v-col>
+
+
+      <v-toolbar>
+        <v-toolbar-items class="pl-8 align-center">
+          <v-chip/>
+        </v-toolbar-items>
+      </v-toolbar>
+
+      <v-col cols="12" class="px-6">
```

Comment:
You don't need to next an additional row in this col. Pull up the below `v-col` with the `v-for` and there you go!

Reply:
Alternatively, what do you think about a grid layout instead of a flex layout for these cards? Would help make the heights consistent (once we enable title wrapping)